source/reference/methods.txt

```diff
@@ -356,6 +356,62 @@ Connection Methods
 
      - Description
 
+   * - :method:`Mongo()`
+
+     - JavaScript constructor to instantiate a database connection from
+       the ``mongo`` shell or from a JavaScript file.
+
+       The :method:`Mongo()` method takes a parameter called
+       ``ClientSideFieldLevelEncryptionOptions`` with the following options:
+
+       .. list-table::
+          :widths: 15 20 40
+          :header-rows: 1
+
+          * - Option
+
+            - Type
+
+            - Description
+
+          * - ``keyVaultClient``
+
+            - :method:`Mongo()` connection object
+
+            - *(Optional)* The MongoDB cluster hosting the key vault
+              collection. Omit to use the current database connection
+              as the key vault host.
```

Comment:
I think this is misleading even in the old shell's documentation. It's not the current database connection (i.e. the one referred to by `db`), it's the newly created `Mongo` object's own one (i.e. the one pointed to by the connection string)

Reply:
I've updated to defer to the old shell docs here and keep this description high level. I've created https://jira.mongodb.org/browse/DOCSP-14537 to address the confusion in the old shell docs, which we will get to in this sprint.